server/data/data-library/src/main/java/org/apache/james/rrt/lib/AbstractRecipientRewriteTable.java

```diff
@@ -167,6 +167,16 @@ public Mappings getMappings(String user, Domain domain, int mappingLimit) throws
         return MappingsImpl.empty();
     }
 
+    private Mapping toMapping(String mappedAddress, Type type) {
+        switch (type) {
+            case Forward:
+            case Group:
+                return MappingImpl.of(type, mappedAddress);
+            default:
```

Comment:
we should _never_ use default but list all enum values everytime to ensure a new type is not handled by accident.